WordPress/Classes/Utility/ReachabilityFacade.h

```diff
@@ -1,34 +0,0 @@
-#import <Foundation/Foundation.h>
```

Comment:
This looked like dead code which I was able to remove when `ReachabilityUtils` was refactored.